docs/src/man/getting_started.md

```diff
@@ -645,6 +645,41 @@ Equivalently, the `in` function can be called with a single argument to create
 a function object that tests whether each value belongs to the subset
 (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+#### Selecting Rows with `filter`
+
+We have seen above how to subset a `DataFrame` to several criteria, involving multiple columns, by supplying a logical vector to the first dimension. For instance, in the following we want to subset to all rows where `x > 2` and where `a == 'c'`:
+
+```jldoctest dataframe
+julia> df = DataFrame(:x => 1:4, :y => "a", :a => 'a':'d', :b => exp(1))
```

Comment:
it would be better to make columns `y` and `b` to contain heterogenous data, or just drop them.

Also - your example was produced using DataFrames.jl 0.21, and now we have DataFrames.jl 0.22, which produces a different display.

Reply:
dropped `b` but not `y` (changed to `rand`). it's important to realize that we filter the _entire_ `DataFrame` in this example, so `y` should be there.